cli/tools/test/mod.rs

```diff
@@ -1326,37 +1326,30 @@ pub async fn check_specifiers(
   )
   .await?;
 
+  let mut module_specifiers = specifiers
+    .into_iter()
+    .filter_map(|(specifier, mode)| {
+      if mode != TestMode::Documentation {
+        Some(specifier)
+      } else {
+        None
+      }
+    })
+    .collect::<Vec<_>>();
+
   if !inline_files.is_empty() {
     let specifiers = inline_files
       .iter()
       .map(|file| file.specifier.clone())
-      .collect();
+      .collect::<Vec<_>>();
```

Comment:
Nitpick: Looks like we could just do `module_specifiers.extend(inline_files.iter().map(|f| f.specifier.clone())` here.